ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestCaseBuilderUtil.java

```diff
@@ -133,11 +133,25 @@ public static String extractSimpleTestName(
                   .filter(Optional::isPresent)
                   .orElse(topicFromStatement.getValueSchema());
           topic = new Topic(topic.getName(), topic.getNumPartitions(), topic.getReplicas(),
-                  keySchema, valueSchema);
+                  keySchema, valueSchema, topic.getKeyFeatures(), topic.getValueFeatures());
 
           return topic;
         });
-        allTopics.putIfAbsent(topicFromStatement.getName(), topicFromStatement);
+        if (allTopics.containsKey(topicFromStatement.getName())) {
+          // If the topic already exists, just add the key/value serde features
+          final Topic existingTopic = allTopics.get(topicFromStatement.getName());
```

Comment:
This is needed, because in some of the tests the statement includes SerdeFeatures like `UNWRAP_SINGLES`, but if the test's json file includes `"topics": [{ ...` then that information isn't included.